### ISSUE TITLE
`?component` param needed to get correct in-editor typing

### ISSUE THREAD

**Issue by beefchimi** [jpkleemans/vite-svg-loader#134]
Without the `?component` param at the end of the `.svg` import, VsCode reports the import as a `string`.

Only by appending the `?component` will VsCode report it as a `FunctionalComponent`:

![import](https://github.com/jpkleemans/vite-svg-loader/assets/643944/fff12144-bcd7-4a21-b6d1-7b5aacbc6174)

I tried adding `defaultImport: 'component'` to the `svgLoader` config and it made no difference.

**Comment by microHoffman** [comment]
Yeah I have similar issue, I've resolved that by manually putting this to my `index.d.ts`:
```ts
declare module '*.svg' {
    import type { FunctionalComponent, SVGAttributes } from 'vue'
    const src: FunctionalComponent<SVGAttributes>
    export default src
  }
```
